docs/language_specification/tokens/tokens.md

```diff
@@ -0,0 +1,2 @@
+There are the following kinds of tokens: newlines, literals, keywords, identifiers, operators, and other separators.</br>
+Blanks, tabs, and comments (collectively, "white space") are ignored except as they serve to separate tokens.
```

Comment:
This file can be removed. Maybe the 2nd sentence can be added to the Whitespace section (if not already explained there).

Reply:
Done. I've restructured the Language Specification part:
- Structure now contains two sections, Lexical and Syntactical, the first pointing to Tokens, and the second to Statements and Expressions.